test_scripts/API/VehicleData/GearStatus/023_App_version_is_more_than_version_gearStatus_in_DB_and_API.lua

```diff
@@ -0,0 +1,64 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0266-New-vehicle-data-GearStatus.md
```

Comment:
Please update the file name to test_scripts/API/VehicleData/GearStatus/023_App_version_is_greater_than_version_gearStatus_in_DB_and_API.lua